src/IdBrokerClient.php

```diff
@@ -312,17 +312,30 @@ public function listUsers(array $fields = null, ?array $search = []): array
      * @param string $type
      * @param string|null $label
      * @param string|null $rpOrigin
+     * @param string|null $recovery_email
      * @return array|null
      * @throws ServiceException
      */
-    public function mfaCreate(string $employee_id, string $type, string $label = null, string $rpOrigin = ''): ?array
+    public function mfaCreate(string $employee_id, string $type, string $label = null, string $rpOrigin = '', ?string $recovery_email = null): ?array
     {
-        $result = $this->mfaCreateInternal([
-            'employee_id' => $employee_id,
-            'type' => $type,
-            'label' => $label,
-            'rpOrigin' => $rpOrigin,
-        ]);
+        if ($recovery_email === null) {
+            $result = $this->mfaCreateInternal([
+                'employee_id' => $employee_id,
+                'type' => $type,
+                'label' => $label,
+                'rpOrigin' => $rpOrigin,
+            ]);
+        }
+        else{
+            $result = $this->mfaCreateInternal([
+                'employee_id' => $employee_id,
+                'type' => $type,
+                'label' => $label,
+                'rpOrigin' => $rpOrigin,
+                'recovery_email' => $recovery_email,
+            ]);
+        }
+        
```

Comment:
Any reason to not do this?

```suggestion
        $result = $this->mfaCreateInternal([
            'employee_id' => $employee_id,
            'type' => $type,
            'label' => $label,
            'rpOrigin' => $rpOrigin,
            'recovery_email' => $recovery_email,
        ]);
```

Reply:
I agree with you @jason-jackson but I don't see that as a reason to withhold PR approval.

Reply:
Or maybe it was for the null issue.